script/publish

```diff
@@ -74,14 +78,14 @@ verbose('git fetch origin')
 verbose('git reset --hard origin/master')
 verbose('git clean -ffxd .')
 
-File.write(version_file, File.read(version_file).gsub(/return '[.0-9]+';/, "return '#{version_string}';"))
```

Comment:
Does this mean the version file will not be updated to the NPM version? 
So if the current version is 1.19.0-20-g1a54723, after running NPM publish, we will have 1.20.0 but the version file will still be 1.19.0-20-g1a54723 right?

Reply:
The `Versioning.ts` file no longer mentions a version at all.

`src/versioning/version.json` will be updated during `build`, right after `barrelize.js` is run.

That JSON file will be copied into the build along with the `tsc`-compiled `Versioning.js`.

`version.json` is not checked into version control at all: it is materialized at build time by looking at the current Git tags.